docs/static/arcsight-module.asciidoc

```diff
@@ -38,30 +48,16 @@ the Smart Connector directly.
 
 image::static/images/arcsight-diagram-smart-connectors.svg[ArcSight Smart Connector architecture]
 
-[[arcsight-requirements-smartconnector]]
-===== Requirements
+Smart Connector has been configured to publish ArcSight data (to TCP port `5000`) using the CEF syslog 
+destination.
 
-* These instructions assume you have part of the Elastic Stack (Logstash, Elasticsearch,
-Kibana) already installed. The products you need are
-https://www.elastic.co/downloads[available to download] and easy to install. The
-Elastic Stack 5.6 or higher is required for this module.
-* The Elastic Stack is running locally with default ports exposed, namely
+NOTE: Logstash, Elasticsearch, and Kibana must be running locally with default ports exposed, namely
 Elasticsearch as "localhost:9200" and Kibana as "localhost:5601". Note that you can also run 
 Elasticsearch, Kibana and Logstash on separate hosts to consume data from ArcSight.
-* Smart Connector has been configured to publish ArcSight data (to TCP port `5000`) using the CEF syslog 
-destination.
 
 [[arcsight-instructions-smartconnector]]
 ===== Instructions
 
-. {ref}/installing-xpack-es.html[Install X-Pack on Elasticsearch] and then start
-Elasticsearch.
-
-. {kibana-ref}/installing-xpack-kb.html[Install X-Pack on Kibana] and then start
-Kibana.
-
-. {logstash-ref}/installing-xpack-log.html[Install X-Pack on Logstash], which
-includes the Logstash ArcSight module.
 . Start the Logstash ArcSight module by running the following command in the
 Logstash install directory with your respective EB host and port:
```

Comment:
@acchen97   These instructions are for Smart Connector.  Step 1 in original content references "EB host."  Is that correct? Or should it be Smart Connector?

Reply:
@karenzone yep typo thanks for taking care of it.